src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapSessionOptions.Linux.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.ComponentModel;
```

Comment:
Are all the new napsaces needed here and in the other .cs file?

Reply:
Oh, right. `System.ComponentModel` is needed for `InvalidEnumArgumentException` in both files. The other two in `LdapSessionOptions.Linux.cs` are not needed anymore. I'll remove them.